app/handlers/uploadVideo.py

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/env python
 
 import tornado.web
+from tornado.web import stream_request_body
+from tornado.httputil import parse_multipart_form_data
+import trafficcloud.api as api
```

Comment:
Try to remove all calls to api and trafficcloud folder as a whole. Perhaps move any function needed to a file in  traffic_cloud_utils or create a new function in your working file.